spec/error_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+
+module Stitches
+  describe Error do
+    describe '#initialize' do
+      context 'required params are set' do
+        subject { described_class.new(code: anything, message: anything) }
```

Comment:
Thanks for writing these tests!

For this paticular one, I don't know if we want to bind this tightly to the implementation.  I think you could do something like

```ruby
expect {
  described_class.new(code: anything, message: anything)
}.not_to raise_error
```